src/K9OCRS/ClientApp/src/areas/applications/Catalog/Classes.js

```diff
@@ -54,13 +54,18 @@ const Classes = props => {
         <p>{classDetail.description}</p>
         <h5>Requirements</h5>
         {classDetail.requirements ? <p>{classDetail.requirements}</p> : <p>This class has no requirements</p>}
-        {loading ? <Spinner /> : <Table
+        <h5>Sections</h5>
+        {classDetail.sections?.length > 0 ?  
+        <div>Choose one of the sections below to start the application process!</div>
+        :  <div>There are currently no sections available for this course. Please check back soon!</div>}
+        
+        {loading ? <Spinner /> : (classDetail.sections?.length > 0)? <Table
             columns={sectionColumns}
             data={classDetail.sections}
             pageSize={12}
             footnotes={['* This is the usual meeting time, but it may vary']}
             withPagination
```

Comment:
Avoid nesting Ternaries inside other Ternaries. It makes them awkward to read/scan over. Prioritize readability over lines of code.

if you want to learn more on this, [here's an article on it](https://www.rithmschool.com/blog/using-the-javascript-ternary-operator#:~:text=Best%20Practice%3A%20Don%27t%20Use%20Nested%20Ternaries)
```suggestion
        { loading ? <Spinner /> : null }
        { (!loading && classDetail.sections?.length > 0) ? (
          <Table
              columns={sectionColumns}
              data={classDetail.sections}
              pageSize={12}
              footnotes={['* This is the usual meeting time, but it may vary']}
              withPagination
          />
```